package.json

```diff
@@ -10,6 +10,7 @@
     "start": "node ./src/cli/index.js",
     "test": "jest",
     "pretest": "eslint .",
+    "test:diff": "jest --onlyChanged --no-cache -u --detectOpenHandles -o --findRelatedTests --passWithNoTests --collectCoverage false",
```

Comment:
Awesome way to run tests.
I'm in doubt only about flags `--detectOpenHandles` `-u` and `-o` (this last one is just an alias of --onlyChanged)